src/kiota/Handlers/Plugin/GenerateHandler.cs

```diff
@@ -1,15 +1,26 @@
 ﻿using System.CommandLine;
+using System.CommandLine.Hosting;
 using System.CommandLine.Invocation;
+using System.Diagnostics;
 using System.Text.Json;
+using kiota.Extension;
+using kiota.Telemetry;
 using Kiota.Builder;
 using Kiota.Builder.Configuration;
 using Kiota.Builder.WorkspaceManagement;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 
 namespace kiota.Handlers.Plugin;
 
 internal class GenerateHandler : BaseKiotaCommandHandler
 {
+    private readonly KeyValuePair<string, object?>[] _commonTags =
+    [
+        new(TelemetryLabels.TagGenerationOutputType, "plugin"),
+        new(TelemetryLabels.TagCommandName, "generate"),
+        new(TelemetryLabels.TagCommandRevision, 2)
```

Comment:
Just to confirm is this value supposed to be a `2`? Same case for the Client `GenerateHandler`?

Reply:
I wanted to separate between the old `generate` command vs the new experimental one in the telemetry